### DISCUSSION TITLE
Do I need Syncthing on the device or not?

### DISCUSSION THREAD

**Discussion by gerroon** [LBF38/obsidian-syncthing-integration#145] [Q&A] [answered]
Hi

Your readme makes it like I do not even need to have ST on the device that this plugin is installed on, as in I can just point it to another device's IP that has ST on it. Is that correct? If so I can install Obsidian and this extension on my IOS device, and point the the IP of the ST device (which is my desktop)  let it sync the content through my Desktop which has Syncthing running on it without having ST running on my IOS device at all (which would be amazing)?

thanks

**Comment by LBF38** [comment]
Hi,
Sorry for the misunderstanding. 

The plugin is only an integration of Syncthing's features in Obsidian. It doesn't contain any Synthing instance nor sync any files. It only makes requests to the Syncthing's REST API. 
Therefore, in that sense, you can totally have a remote Syncthing instance and have the plugin requesting it. You'll have some Syncthing features inside Obsidian (currently working on it, see #10 and #137 for more info. I'll update the plugin when I have more time).

Moreover, the plugin tries to implement other features more useful when syncing using Syncthing and working directly from Obsidian. For example, the plugin can help you manage any conflicting files from Syncthing and resolve them. There is also a status bar feature that will come soon. (see #138).

Hope it helps !
Thx

PS: have a look at #133.

**Reply by gerroon** [reply]
Hi

thanks for the reply. It seems like it is not working because it can't find the CLI access. I am not sure if this is necessary or not? My API access works with the remote instance since it the test returns pong.

**Reply by LBF38** [reply]
You should be able to access it if the REST API responds w/ "pong". However, some features might not yet be available as I need more time to implement them (see #137).
Thank you for your understanding.

**Comment by gerroon** [comment]
Also is the folder access  set up is done? I do not see the Obsidian folder in the Syncthing Web page? Maybe it does not even need to be added to ST? This is a bit confusing since normally One has to add all the endpoints and folders in ST on all others to make it work.

**Reply by LBF38** [reply]
The plugin doesn't come with a Syncthing instance. So, to sync your folder across your devices, you need to have a running Syncthing instance on your device.
This plugin only aims to integrate some useful features when syncing your Obsidian vault with Syncthing.

**Reply by gerroon** [reply]
You said that I can use a remote ST (running on  another device) with API access which is exactly how I set it up on my device.

DeviceA: Obsidian device with the plugin, this device does not have ST running or ST CLI on it.
DeviceB: Syncthing instance

DeviceA is running this plugin with Obsidian, I added DeviceB ST API key and the DeviceB IP. Now what are the next steps to sync? It seems to me that I need to add DeviceA to DeviceB, given DeviceA has no running ST, it is not possible because this Obsidian extension does not support it?

**Reply by LBF38** [reply, marked as answer]
As I said, it will not sync DeviceA because:
1. The plugin doesn't sync any files. It only interacts with the Syncthing REST API.
2. There is no Syncthing instance on DeviceA. So, no files will be synced between the two devices.

You will only be able to see the information of your Syncthing instance from within Obsidian, and have some other features related to Syncthing in Obsidian.

**Reply by gerroon** [reply]
Thanks, that makes sense
